package.json

```diff
@@ -43,7 +43,8 @@
     "src/**/*"
   ],
   "dependencies": {
-    "@evolv/client": "2.4.9-alpha-1d367c2"
+    "@evolv/client": "2.4.9-alpha-1d367c2",
+    "@sentienttechnologies/dom-capture": "^1.0.2"
```

Comment:
I think we're going to need to publish your DOM packages to our public registry on npmjs.org (the `@evolv` scope rather than `@sentienttechnologies`. `@sentienttechnologies` won't be accessible to most consumers.